sections/findify-autocomplete.liquid

```diff
@@ -0,0 +1,101 @@
+{%- assign page_url = content_for_header | split:'"pageurl":"' | last | split:'"' | first | split: request.host | last | replace:'\/','/' | replace:'%20',' ' | replace:'\u0026','&'  -%}
+
+{%- assign query = page_url | split: 'findify-autocomplete&q=' -%}
+{%- assign suggestionArray = query[1] | split: '&suggestion=' | first | split: 'isMobile' | first -%}
+{%- assign suggestions = suggestionArray | split: ',' -%}
+{%- assign productHandlers = query[1] | split: '&suggestion=' %}
+{%- assign isMobile = query[1] | split: '&isMobile=' | last -%}
+{%- assign array = productHandlers[0] | split: ',' %}
+{%- assign suggestionsTemplate = "vertical"-%}
+{%- assign viewAllPosition = "bottom" -%}
+{%- assign findifyAutocompleteContentTitle = settings.findifyAutocompleteContentTitle -%}
+{%- assign findifyAutocompleteContentID = settings.findifyAutocompleteContentID -%}
+{%- assign class = "findify-autocomplete findify-autocomplete-fullscreen" -%}
+{%- assign productCard = '' | append: "findify-product-card" -%}
+
+<div id="findify-autocomplete" class="findify-autocomplete findify-autocomplete-fullscreen hidden">
+<div class="findify-autocomplete-content{% if findifyAutocompleteContentTitle.size > 0 and findifyAutocompleteContentID.size > 0 %}columns-3{% endif %}">
+    <div class="suggestions-wrapper {% if suggestionsTemplate == 'horizontal' %} suggestions-horizontal {% endif %}">
+      <h3>Search Suggestions</h3>
+      {% if suggestions.size > 0 %}
+      <ul>
+        {% for i in suggestions %}
```

Comment:
instead of i put suggestion
something meaningful so anyone who reads it can relate